00.Welcome.md

```diff
@@ -39,3 +39,9 @@ The things that power your Gitter activity feed. These are open source and can b
 ## Activity API
 
 The ability to write to the Activity feed will be coming soon in an open source format.
+
+## Packages
```

Comment:
Hmm, I do agree that it could be better. We use [*integration*](https://gitter.zendesk.com/hc/en-us/articles/200176722-Integrations) to describe the modules for the activity stream in the Gitter UI and I wouldn't want to add confusion to that.